scripts/system_tests_utils.ts

```diff
@@ -56,6 +59,21 @@ const accounts: {
 	},
 ];
 
+export const createNewAccount = async (): Promise<{ address: string; privateKey: string }> => {
+	const acc = createAccount();
+	const clientUrl = getSystemTestProvider().replace('ws://', 'http://');
+	const web3Personal = new EthPersonal(clientUrl);
+	await web3Personal.importRawKey(acc.privateKey.substring(2), '123456');
+	await web3Personal.unlockAccount(acc.address, '123456', 500);
```

Comment:
If you are only creating a test account for other functions, Its not necessary to put account on node's wallet.

Only difference: In case of local wallet account you will need to explicitly sign while sending transactions from that new account.

Reply:
I want to create and unlock an account for each test to avoid nonce problems. It works in GETH but does not work in Ganache. I need to investigate more. So I don't need this helper. but it looks like we need it for the future